client/my-sites/exporter/export-card/advanced-settings.jsx

```diff
@@ -8,7 +8,7 @@ import { connect } from 'react-redux';
  * Internal dependencies
  */
 import PostTypeOptions from './post-type-options';
-import SpinnerButton from './spinner-button';
+import SpinnerButton from 'components/button/spinner-button';
```

Comment:
Few issues with this change:

- Why not create a separate `components/spinner-button` ?
- The contained `className` (`export-card__spinner-button`) does not adhere to [`wpcalypso/jsx-classname-namespace`](https://github.com/Automattic/eslint-plugin-wpcalypso/blob/master/lib/rules/jsx-classname-namespace.js)